src/main/java/mcp/mobius/betterbarrels/BetterBarrels.java

```diff
@@ -69,7 +69,7 @@ public static void debug(String msg) {
 
     /* CONFIG PARAMS */
     private static Configuration config = null;
-
+    public static boolean DisableStacking;
```

Comment:
nitpick: Please, use `lowerCamelCase`  and more descriptive name. Current name sounds like this config disables stacking of items inside barrel.
```suggestion
    public static boolean disableDollyStacking;
```